packages/schemas/tables/passcodes.sql

```diff
@@ -2,7 +2,7 @@ create type passcode_type as enum ('SignIn', 'Register', 'ForgotPassword');
 
 create table passcodes (
   id varchar(128) not null,
-  session_id varchar(128) not null,
+  interaction_jti varchar(128) not null,
```

Comment:
This is a postgres noob question: do we need to drop the table before create it with new columns?

Reply:
That is a question about migration, we have a linear project, that's hold it until v1 shipped, focus on code now.

Reply:
OK, got it.